app/states/texas/texas_downloader.py

```
@@ -7,90 +9,271 @@
+                                continue
+                        
+                        _data[k] = df.to_dicts()
+                    except Exception as e:
+                        ic(f"Error processing category {k}: {e}")
+                        _data[k] = []
+            
+            cls.data = _data
+        return cls.data
+
+    @classmethod
+    def dataframes(cls):
```

Ruff (0.14.7)

Missing return type annotation for classmethod `dataframes`

(ANN206)